tests/ui/hygiene/legacy_interaction.rs

```diff
@@ -20,9 +19,9 @@ extern crate legacy_interaction;
 mod def_site {
     // Unless this macro opts out of hygiene, it should resolve the same wherever it is invoked.
     pub macro m2() {
-        ::legacy_interaction::m!();
+        ::legacy_interaction::m!();         //~ ERROR cannot find function `g` in this scope
```

Comment:
This doesn't look correct.
`g` is an item, so it should be found at call site, not at def site, because `m!` is not a macro 2.0.

`macro_rules` use https://doc.rust-lang.org/stable/proc_macro/struct.Span.html#method.mixed_site so only local variables and labels are looked up at definition site (and `$crate`, but we are not talking about it here).